### DISCUSSION TITLE
WWETH not transferred

### DISCUSSION THREAD

**Discussion by 0x1k** [wormhole-foundation/wormhole#355] [Problems / bugs] [answered]
Hi,

I swapped eth to wweth using wormhole and later did two transactions to swap wweth back to eth. The first transaction was a test transaction and it worked fine but the second seems to not have gone through. I saw this can be solved by manually submitting something on the contract but I'm not sure. Can anyone help me please?

Test tx- https://explorer.solana.com/tx/4Ga6z9MivqodhkQ6qhLmRCEKb2hw9BbJdpM7bH1RzZ4F35GFEJ9ZJtrev8KmUFZwJ8XWrGiv7aT2HF5KC55E9XDA

Stuck tx- https://explorer.solana.com/tx/2BWEE55a6iQUkWuvZfJbdCU6haREA45MxqzfofVa89hbUmA7GEbyE5UFyqcsgEeHqg6CFfiFgua62BiXg5oGTDh5

**Comment by leoluk** [comment, marked as answer]
The transaction was successfully processed by the Wormhole network, but the client-side transaction submission may have failed. Try clearing your cache and reloading wormholebridge.com to see if the transaction can be completed there. Make sure both wallets are connected.

If wormholebridge.com doesn't work, you can manually submit the signed VAA bytes below to the submitVAA method on the Wormhole contract: https://etherscan.io/address/0xf92cd566ea4864356c5491c177a430c222d7e678#writeContract

---

Digest: bbcd5043898856533d49859bd280db0435ee6096d7fecfea4270e6cd7dcb72d1

Signed VAA bytes:

> 0x01000000010d009234c77e1c6225ed4c6feef5eeb8f5fc23df5623a21e0fb1322fb2849d7977d922603950fdd10e554f3d3bb4b71d4cc2dcb4934d93c32443e343e79a121d7a230001dc4b14e10abb1eef1024043265ce0274054daa730ec6f44f2d3d849bd53cdc04208297427c9d1aa21bf67734affa63abe33c4c3af11509d7274b43a2ba48aa240103ad7d6db38b0e9ffd0170ebb3cb9c0986c0f78a7d413c34854630657d6c1c7b907a04750f2ac6dad0397c58b47cfbb20a0f4970273bbb895e0a1173d93b7f993d0005d7bae49cf4c8d867086b2d8641c4ed52b46e566587f453466c6aec675a76136631c3ff5277e971bcc5759eca3361ed868f3a63250eedbaa028d2d79d17e631700007ffc74387bc17e27cd7e787171ec43bd6afdba55b70547d9e0726e0296259d0e50b82ef76c0107ea06bf7ae3ec4e70666a79220fa69d14431acb247a91b676775000882e469ce06419f61c10583b3cb67e962a6aa7acdcb5c17cd1a38be905042a17d7ddf1bc219bbffb0dadff670810439767ea231dabba851d7949bbc12bc193be20009dd83c81f101be1a40376b2edbce830452256587a2ddb21b380b4270ec4a4b08967432c3f01d14a8c462633a4569e5fb155526aa8c81c349f3b7ce387b3973f1c000ada3387fa368dcecb9f18e78c8bbd929c8115ca9f2d98c70fc494e3b8528e4e4f57db04329bd9c513a4427b43cd5ef74918187c669e486d4afc3800100ccc691e000b4892cc3e3626ae2cd72b83240131dc28645935ca637817e564a345e61c757ad55c7d4f8dd5e8744a2bf3b74f67680660362acd61519f65346b54fe9b83823988010d2db3a5334699b9ced80b9f04679cd11ba0fe19ba14d2e8ea6870fb1afd68aa476ea3e6455321c2633de8b806e9be7c7ee354b40a72aaa7cb845593752d0d524b0010aaf85dca44e3fdde9c432506d158a0f84fc78454221241de418e663857c0a3a5509cba055c84935b8d719b6fbde384ebe9cab426929acd59d70ba5e01619053c0111bea81440d9148ae93b20f977bac307f01a09bbbb423fd70f25b100a602e30b3378c07519f28363a3d5914a6e8ac690b86fb233bc05b73b2cbfe5c79803da04590012d51189469ca495daee7acb073d8362c339825cd176e52f04f4115e7a1cede25a62d951ce27277b74b462db72132dce4168476cb7aa1d62bf64203a444f3f69d001612b33c3100000385a01026c1df7c23a40377a01ccff67ba57d31677befdd23178539fd4ce0e5e2cc2d7120000000000000000000000008967ead56e5a800a4fa60477b79993db5180548b02000000000000000000000000c02aaa39b223fe8d0a0e5c4f27ead9083c756cc209000000000000000000000000000000000000000000000000000000003d648d80

**Reply by 0x1k** [reply]
Thanks! :)

**Reply by leoluk** [reply]
Whoop :) Glad to be of help!
